### ISSUE TITLE
Failed to launch pod with network

### ISSUE THREAD

**Issue by fabiand** [mpolednik/linux-vfio-k8s-dpi#8]
I attached a bridge to a pod:

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: demo-pod
spec:
  containers:
  - name: nginx
    image: nginx:latest
    resources:
      limits:
        bridge.network.kubevirt.io/mybridge: 1
```

but it fails with:

```
2018-02-19 21:44:02 +0100 CET   2018-02-19 21:44:02 +0100 CET   1         demo-pod.1514d4e22fea75ad   Pod       spec.containers{nginx}   Normal    Pulled    kubelet, minikube   Successfully pulled image "nginx:latest"
2018-02-19 21:44:02 +0100 CET   2018-02-19 21:44:02 +0100 CET   1         demo-pod.1514d4e234b0687f   Pod       spec.containers{nginx}   Normal    Created   kubelet, minikube   Created container
2018-02-19 21:44:02 +0100 CET   2018-02-19 21:44:02 +0100 CET   1         demo-pod.1514d4e237b96a1c   Pod       spec.containers{nginx}   Warning   Failed    kubelet, minikube   Error: failed to start container "nginx": Error response from daemon: linux runtime spec devices: error gathering device information while adding custom device "/tmp/deviceplugin-network-bridge-fakedev": no such file or directory
```

I wonder where fakedev comes from.

**Comment by phoracek** [comment]
fakedev is created by the plugin itself https://github.com/kubevirt/kubernetes-device-plugins/blob/master/pkg/network/bridge/plugin.go#L65. will look into it.

**Comment by phoracek** [comment]
The patch should make it possible to debug the problem, not fix it.

**Comment by phoracek** [comment]
@fabiand could you please try again with new dp image and attach the whole log?

**Comment by fabiand** [comment]
I can not retrieve the DP pod's logs due to #7.

**Comment by fabiand** [comment]
Pod creation is still failing with

```
2018-02-22 09:10:26 +0100 CET   2018-02-22 09:09:51 +0100 CET   2         demo-pod.151597783411328e   Pod       spec.containers{nginx}   Normal    Pulling   kubelet, minikube   pulling image "nginx:latest"
2018-02-22 09:10:29 +0100 CET   2018-02-22 09:10:26 +0100 CET   2         demo-pod.15159780362a70c4   Pod       spec.containers{nginx}   Normal    Pulled    kubelet, minikube   Successfully pulled image "nginx:latest"
2018-02-22 09:10:29 +0100 CET   2018-02-22 09:10:26 +0100 CET   2         demo-pod.151597803ad3c683   Pod       spec.containers{nginx}   Normal    Created   kubelet, minikube   Created container
2018-02-22 09:10:29 +0100 CET   2018-02-22 09:10:26 +0100 CET   2         demo-pod.151597803d3327bc   Pod       spec.containers{nginx}   Warning   Failed    kubelet, minikube   Error: failed to start container "nginx": Error response from daemon: linux runtime spec devices: error gathering device information while adding custom device "/tmp/deviceplugin-network-bridge-fakedev": no such file or directory
2018-02-22 09:10:40 +0100 CET   2018-02-22 09:09:51 +0100 CET   3         demo-pod.151597783411328e   Pod       spec.containers{nginx}   Normal    Pulling   kubelet, minikube   pulling image "nginx:latest"
```

**Comment by fabiand** [comment]
Was fixed by #12
